### PR TITLE
leave/join message array size changed

### PR DESCRIPTION
**Pull request by ghost**
To not unnecessarily waste memory

### CODE REVIEW

File: gamemodes/roam.pwn [hunk 1]
```diff
@@ -289,7 +289,7 @@ public OnPlayerConnect(playerid)
 public OnPlayerDisconnect(playerid, reason)
 {
 	new pName[MAX_PLAYER_NAME];
-	new leaveMessage[128];
+	new leaveMessage[27 + MAX_PLAYER_NAME + 1];
```
Comment (Acygol) [inline gamemodes/roam.pwn]: Test review

Comment (Brian3898) [reply gamemodes/roam.pwn]: Test 1 2 3

Comment (Acygol) [reply gamemodes/roam.pwn]: It's a good test, yes.

**Review by Brian3898 — COMMENTED** [review]
Good change overall

**Pull request merged**